packages/tools/pie-icons-react/scripts/build.mjs

```diff
@@ -62,6 +55,9 @@ async function build () {
     // check that the /icons directory exists, if not create it
     await checkDirExists(ICONS_DIR);
 
+    // open a write stream to index.tsx
+    const indexFile = fs.createWriteStream(filePath);
```

Comment:
The write stream should be created after the directory is created, in the previous line.